graf3d/eve7/inc/ROOT/REveElement.hxx

```diff
@@ -33,10 +33,20 @@ class TGeoMatrix;
 #include <string> // string
 #include <vector> // vector
 
+/*!
```

Comment:
I am confused.  Why do we have the file both as a separate (unused) file and an embedded fragment?

Reply:
If I use include statement here, I need to be sure that include path like `$ROOTSYS/io/io/res` will be always provided - 
in cling, in ACLiC, in user-derived code. 
After this PR I want to change this in master - one should use normal include from the beginning and it should be placed 
in `$ROOTSYS/include/nlohmann` if no other standard location is provided. But this just a fix for current issue

Reply:
Fair enough